### PR TITLE
fix(upgrade): Add the latest changes of retheme project

### PR DESCRIPTION
**Pull request by anagstef**
## Description

- Created a new breaking changes section for appearance prop variables.
- Added the latest localization changes.

## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

File: packages/upgrade/source/guide-generators/v5/retheme/index.js [hunk 1]
```diff
@@ -5,14 +5,22 @@ const cwd = 'v5/retheme';
 async function generate() {
   const data = (await import(`../../../versions/v5/index.js`)).default.nextjs;
 
+  const appearancePropBreakingChanges = {
+    title: 'Breaking Changes to appearance variables',
+    content: `\nSeveral appearance variables have been removed or renamed. If you were using these variables in your application, you will need to update your code to use the new variables.
+
+    - The \`fontSmoothing\` variable has been removed.
+    - The \`shadowShimmer\` variable has been removed.
+    - The \`colorAlphaShade\` variable has been renamed to \`colorNeutral\`.\n`,
```
Comment (jescalan) [inline packages/upgrade/source/guide-generators/v5/retheme/index.js:14]: Is there no realistic way we can regex scan for these? Assuming so but just want to double check

File: packages/upgrade/source/versions/v5/common/removed-localization-keys.md [hunk 2]
```diff
@@ -111,7 +110,6 @@ const deletedKeys = [
   'userProfile.start.connectedAccountsSection.title__conectionFailed',
   'userProfile.start.connectedAccountsSection.title__connectionFailed',
   'userProfile.start.connectedAccountsSection.title__reauthorize',
-  'userProfile.start.connectedAccountsSection.subtitle__reauthorize',
```
Comment (jescalan) [inline packages/upgrade/source/versions/v5/common/removed-localization-keys.md:114]: These were re-added right?

**Review by jescalan — APPROVED** [review]
Thank you, this is great!

File: packages/upgrade/source/guide-generators/v5/retheme/index.js (hunk 1 above)
Comment (anagstef) [reply packages/upgrade/source/guide-generators/v5/retheme/index.js:14]: @jescalan Oh, I think we can! I will add a commit.

File: packages/upgrade/source/versions/v5/common/removed-localization-keys.md (hunk 2 above)
Comment (anagstef) [reply packages/upgrade/source/versions/v5/common/removed-localization-keys.md:114]: Yes!

File: packages/upgrade/source/guide-generators/v5/retheme/index.js (hunk 1 above)
Comment (jescalan) [reply packages/upgrade/source/guide-generators/v5/retheme/index.js:14]: If so, can be written up as a markdown file inside `versions/v5/common` folder and referenced from there using `singleItem` 😁

**Review by jescalan — APPROVED** [review]
Amazing, thank you! Sorry about the big merge conflict that you inevitably had to work through due to the name change 😅

**Pull request merged**